README.md

```diff
@@ -2,8 +2,9 @@
 
 A collection of helper tools in the form of Java libraries optionally
 controlled through command line arguments. This project is the successor of an
-internal project called "base." The main aim of this project is to provide
-lightweight integration behind a common flags implementation.
+internal project called "base." Secondbase is open sourced version of "base."
```

Comment:
Previous comment still stands: Since this is not in the context of TelenorDigital/comoyo; "internal" makes no sense without describing what it's internal to.

And: Secondbase is an open sourced version of "base".